src/utils/simple_fetch.js

```diff
@@ -14,8 +14,8 @@ import { Error } from 'jsonapi-serializer'
  * @return {object} The parsed JSON, status from the response
  */
 function parse (response) {
-  return new Promise((resolve) => response.text()
-    .then(text => {
+  return new Promise(resolve =>
+    response.text().then(text => {
```

Comment:
I personally think we should use the 1-line syntax only when it's covering a _simple_ `return` statement. This is more complex than that, so we should treat it as a "normal" function.

```jsx
return new Promise((resolve) => {
  response.text().then((text) => {
    // Blaaah.
  })
})
```